eotsmanager/localmanager.go

```diff
@@ -194,6 +199,20 @@ func (lm *LocalEOTSManager) CreateRandomnessPairList(fpPk []byte, chainID []byte
 }
 
 func (lm *LocalEOTSManager) SignEOTS(fpPk []byte, chainID []byte, msg []byte, height uint64, passphrase string) (*btcec.ModNScalar, error) {
+	record, found, err := lm.es.GetSignRecord(height)
```

Comment:
It's not safe to use data directly from db. Better to have a counter part of the db type to have validated data